pkg/util/workload/workload.go

```diff
@@ -0,0 +1,17 @@
+package workload
+
+import (
+	"context"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	kueue "sigs.k8s.io/kueue/apis/kueue/v1beta1"
+)
+
+func RemoveWorkloadFinalizer(ctx context.Context, c client.Client, wl *kueue.Workload) error {
```

Comment:
Should we move this function to the `workload` (pkg/workload) package?

Reply:
Done in f0690dcf